0001-Force-gtk2.patch

```diff
@@ -0,0 +1,36 @@
+From d0c153b4d3cccc87535042015a8e4e7f630c5ba6 Mon Sep 17 00:00:00 2001
+From: Robin Lee <cheeselee@fedoraproject>
+Date: Wed, 24 Jan 2018 18:15:58 +0800
+Subject: [PATCH] Force gtk2
```

Comment:
Curious how bad the Gtk3 support is, because Gtk3 is a lot better of a UX with Flatpak since it supports theming.